tslint.json

```diff
@@ -0,0 +1,61 @@
+{

```

Comment:
This file should have 2 space indentation instead of 4 space. @hweeks can you handle this in your next PR?